api/config.js

```diff
@@ -20,20 +23,26 @@ if (!isProduction && !isDevelopment) {
 
 if (!secret) {
 	if (isProduction) {
-		logger.error("the secret was not provided");
 		logger.error("set HACKILLINOIS_SECRET to a secure, random string");
-
 		process.exit(1);
 	}
 
 	secret = 'NONE';
 }
 
+if (!mailApiKey) {
+	if (isProduction) {
+		logger.error("set HACKILLINOIS_MAIL_KEY to the API key");
+		process.exit(1);
+	} else {
+		logger.warn("the mail server API key was not set");
+		logger.warn("some mail service functionality may malfunction");
+	}
+}
+
 if (!superuserEmail) {
 	if (isProduction) {
-		logger.error("the superuser email was not set");
 		logger.error("set HACKILLINOIS_SUPERUSER_EMAIL to the admin email");
```

Comment:
Error String might be more clear if its was something like "set ENV variable HACKILLINOIS_SUPERUSER_EMAIL to a valid email" or "set ENV variable HACKILLINOIS_SUPERUSER_EMAIL to the email of the admin" The statement "the admin email" implies that there might be some variable or something out there.